spec/login_helper.rb

```diff
@@ -18,3 +18,10 @@ def login_sarah_fewposts
   fill_in :user_password, with: 'sf1234'
   click_button 'Log in'
 end
+
+def login_clare_noposts
```

Comment:
Don;t think this one was actually used in a test but good to have the helper for future